test/k8sT/Services.go

```diff
@@ -846,6 +986,13 @@ var _ = Describe("K8sServicesTest", func() {
 					res.ExpectFail("NAT entry was not evicted")
 				})
 			})
+
+		// Net-next and not old versions, because of LRU requirement.
+		// TODO: Skip if no IPv4 or if fragment support is disabled
```

Comment:
These services tests I think currently always enable IPv4, and disabling fragments support is not currently configured in CI. Are there any actionable items on this TODO?

Reply:
I wasn't sure, hence the TODO. But if we're good with the code and CI doesn't have a chance to run without IPv4/fragment option, I'll remove the comment.